cerberus/api.py

```diff
@@ -11,7 +11,6 @@
 from django.urls import path
 
 # Third Party
-from dateutil import parser
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Good job on removing unused imports.

Removing the unused 'parser' import from 'dateutil' helps in keeping the codebase clean and reduces unnecessary dependencies.